### ISSUE TITLE
 transfering golem shells as a antag doesnt transfer antag datum

### ISSUE THREAD

**Issue by Yogbot-13** [yogstation13/Yogstation#17933]
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: be a golem, transfer into a empty shell by clicking on it, loose antag status

 Submitted by: ImmortalPlagueRat (605167460450304019)

**Comment by MajManatee** [comment]
Yeah you are technically becoming a new golem, not transferring into it.
